frontend/src/views/ImageUploadView.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+
```

Comment:
Add the needed import statements for the material-ui-dropzone to work

```suggestion
import { DropzoneDialog } from "material-ui-dropzone";
import Button from "@material-ui/core/Button";
```